src/pages/AddServers.js

```diff
@@ -87,7 +87,28 @@ class AddServers extends BaseUpdateWizardPage {
       });
   }
 
-  assembleProcessPages = () => {
+  assembleInstallProcessPages = () => {
+    let pages = [];
```

Comment:
Instead of defining `pages` to be an empty array and using `push` to append to it, it would be simpler to just return a static array, i.e.:
```
return ([{
  name: 'SelectedInstallOS',
  component: AddServersPages.SelectInstallOS
},{
  name: 'ProcessInstallOS',
  component: AddServersPages.ProcessInstallOS
},{
  name: 'CompleteInstallOS',
  component: AddServersPages.CompleteInstallOS
});
```

Reply:
ok will update